tremor-script/lib/axiom.tremor

```diff
@@ -0,0 +1,61 @@
+### # Tremor language Axiom integration.
```

Comment:
Optional. Scoping to `integrations::axiom` might make more sense given other integrations are likely and encouraged.

Reply:
I was thinking about that too, then I noticed integrations we have are also top level, so I went with the same model we used there so it's a uniform interface for users.

I think it's also nice, eventually we should have some kind of package manger and doing `targo install axiom` would feel a lot more natural then `targo install integrations::axiom`

Reply:
Ok 👍 - extra comedy points for `targo install` 🤡